include/extractor/profile_properties.hpp

```diff
@@ -10,10 +11,17 @@ namespace extractor
 
 struct ProfileProperties
 {
+    static const constexpr std::size_t MAX_WEIGHT_NAME_SIZE = 256;
```

Comment:
This is a missdirection as the maximum size is 255, actually. Should be 255 and array should have size `MAX + 1`